env/requirements_pytorch.yml

```diff
@@ -0,0 +1,18 @@
+name: data-science-cop-torch
+channels:
+  - defaults
```

Comment:
What I've usually found and been told is that if using conda-forge as a channel, one shouldn't include defaults as a channel, as one can sometimes get conflicts between packages on different channels (something I have experienced before).

Reply:
This inclusion of defaults with conda-forge seems to be the case for all existing environments in the repository: should they be updated too?

Reply:
Yes, they should! Would you mind removing the default channel from the other environments and adding them to this commit?

Reply:
Resolved [1eedbb9](https://github.com/MetOffice/data_science_cop/commit/1eedbb9fdb998c1e945ff19b17d9585eeecca756)